### PR TITLE
UITableView cells: use UITableViewCellStyleSubtitle for readability.

### PR DESCRIPTION
**Pull request by oryonatan**
Previously we used UITableViewCellStyleValue1 which causes long strings
to be truncated, the UITableViewCellStyleSubtitle style allows longer
strings to be shown.

### CODE REVIEW

File: FBTweak/_FBActionTweakTableViewCell.m [hunk 1]
```diff
@@ -31,7 +31,7 @@ @implementation _FBActionTweakTableViewCell
 
 - (instancetype)initWithStyle:(UITableViewCellStyle)style
               reuseIdentifier:(NSString *)reuseIdentifier {
-  if ((self = [super initWithStyle:UITableViewCellStyleValue1 reuseIdentifier:reuseIdentifier])) {
+  if ((self = [super initWithStyle:UITableViewCellStyleSubtitle reuseIdentifier:reuseIdentifier])) {
```
Comment (barakwei) [inline FBTweak/_FBActionTweakTableViewCell.m]: don't change this one

File: FBTweak/_FBTweakColorViewControllerHexDataSource.m [hunk 2]
```diff
@@ -77,7 +77,7 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
  */
 - (_FBEditableTweakTableViewCell *)hexCell
 {
-  _FBEditableTweakTableViewCell *hexCell = [[_FBEditableTweakTableViewCell alloc] initWithStyle:UITableViewCellStyleValue1
+  _FBEditableTweakTableViewCell *hexCell = [[_FBEditableTweakTableViewCell alloc] initWithStyle:UITableViewCellStyleSubtitle
```
Comment (barakwei) [inline FBTweak/_FBTweakColorViewControllerHexDataSource.m]: don't change this one

**Comment by barakwei** [comment]
Fix, squash and rebase. This repo desn't let you merge unless the branch is fully rebased.

**Comment by oryonatan** [comment]
round

**Comment by barakwei** [comment]
Fix commit msg & PR title.

**Pull request merged**